src/App.js

```diff
@@ -51,6 +52,8 @@ class App extends Component {
                key={item.id}            // Key - to help React with performance
 
                // Additional props (event callbacks, etc.) can be passed here
+               type={item.type}
+               score={this.onItemClicked}
```

Comment:
This doesn't indicate to the reader that this is a function! a better name will indicate that it is a callback in the name!